### DISCUSSION TITLE
[SOLVED] Blocking Direct Messages?

### DISCUSSION THREAD

**Discussion by ajburch92** [jagrosh/MusicBot#748] [Questions]
Hi all I think I have a simple error. I was able to get the bot loaded in my server, but get a blocking direct messages response when trying to send any commands ? Can someone help me fix this setting? Thank you - I have attached an image as well.  

![musicboterror](https://user-images.githubusercontent.com/8151812/131395092-72cbbc76-3f8d-42de-93fa-4e8f5a08db5d.PNG)

**Comment by jagrosh** [comment]
You either had the bot blocked, or had disabled DMs in that server.
